### PR TITLE
LRDOCS-4091, Forms and Workflow Integration

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-4091, ready for review, Renato.

### CODE REVIEW

**Comment by natocesarrego** [comment]
Hi @rbohl,

Thanks for update the comment in Jira and send this pull to me. 😃 ASAP I'll review it.

Best.

File: discover/portal/articles/04-business-productivity/01-forms/03-forms-and-workflow.markdown [hunk 1]
```diff
@@ -70,6 +70,30 @@ Entries screen.](../../../images/forms-view-entries-status.png)
...
+- Prioritize families who've never visited The Lunar resort before.
+- Exclude anyone who's on the resort's naughty list, appropriately titled "The
+    Dark Side of the Moon".
+
+To compile a list of finalists, each form entry should go through a workflow
+with one review task. The Single Approver workflow that ships with @product@
+works nicely for this.
+
+After creating the form in the above image, assign the Single Approver workflow
+to it in the form's Settings window, as described in the last section. Publish
```
Comment (natocesarrego) [inline discover/portal/articles/04-business-productivity/01-forms/03-forms-and-workflow.markdown:92]: Hey @rbohl,

This paragraph says the Single Approver can be configured in the last section, however the section which refers this configuration is the one before the last.

**Comment by natocesarrego** [comment]
Hey @rbohl,

I've liked a lot the idea of include a gif in this documentation, because, in my opinion, it helps very much the user comprehension! :) But, I was expecting the gif include the part of approve a form entry too, once it'll allow the user see the whole Workflow of a form entry move from pending to approved. What do you think about include this part?

In addition, the rest of the documentation is good to go to me. 😄 🎉 

Best.

File: discover/portal/articles/04-business-productivity/01-forms/03-forms-and-workflow.markdown (hunk 1 above)
Comment (rbohl) [reply discover/portal/articles/04-business-productivity/01-forms/03-forms-and-workflow.markdown:92]: Thanks Renato, I'll change this.

**Comment by rbohl** [comment]
Hey Renato, I'll try to make a GIF that highlights the approval process as well. It's going to add length to a GIF that's already a bit long, but I'll see if I can make it work. Thanks for the suggestion!

**Comment by natocesarrego** [comment]
One more thing, @rbohl, when this documentation get ready, do you plan to add it after the [https://dev.liferay.com/discover/portal/-/knowledge_base/7-0/creating-advanced-forms](https://dev.liferay.com/discover/portal/-/knowledge_base/7-0/creating-advanced-forms)?

I've seen that the documentation of creating-advanced-forms already has a section mentioning the Workflow configuration, thus I understand that this documentation being reviewed will replace this section, am I right? I just would like to be updated about the Forms documentation. 😁 

Thanks.

**Comment by rbohl** [comment]
Good idea. I honestly forgot that we had a section on workflow in the Advanced Forms article, but we should tear it out in favor of this more detailed article.

**Pull request closed without merging**